reconstruction.py

```diff
@@ -352,10 +394,12 @@ def reconstruct(self,evrange=(-1,-1,-1)):
     # override the default, if given by option
     if options.tmpdir:
         tmpdir = options.tmpdir
-
-    os.system('mkdir -p {tmpdir}/{user}'.format(tmpdir=tmpdir,user=USER))
```

Comment:
no, user e' necessario per girare al LNGS, perche' non e' detto che uno abbia i permessi su /tmp (mentre su alcune macchine, al momento del login, viene creato /tmp/$USER

Reply:
Per quanto riguarda la cartella dei file, in realtà così come l'ho scritto, se uno ha già dei file in locale o vuole scaricarli in una cartella specifica dove ha permessi può farlo (penso anche solo a quando faccio dei test in locale sul mio pc). Se invece non viene specificata alcuna cartella quando si lancia il reconstruction, si compota esattamente comen prima andando su /tmp/ e creando la cartella user.. non va bene?

Reply:
OK